R/dm.R

```diff
@@ -272,6 +272,7 @@ print.dm <- function(x, ...) {
 
 #' @export
 `[[.dm` <- function(x, name) {
+  if (class(name) == "integer" || class(name) == "numeric") abort_no_numeric_subsetting()
```

Comment:
```suggestion
  if (is.numeric(name)) abort_no_numeric_subsetting()
```

`class()` can also have length > 1.